lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -47,7 +47,7 @@ def purchase(money, payment, options = {})
 
       def authorize(money, payment, options = {}, capture = false)
         post = build_auth_request(money, payment, options)
-        post[:processingInformation][:capture] = true if capture
+        post[:processingInformation][:capture] = true if capture && !payment.is_a?(String)
```

Comment:
hm, it seems like the purchase remote tests are failing due to this not being added - my suggested change may have been incorrect, but I'm not sure why it would be.

Reply:
oops - it should be `payment.is_a?(String)` 😆 sorry!